pkg/networkservice/chains/nsmgr/server_test.go

```diff
@@ -140,3 +148,114 @@ func TestNSMGR_LocalUsecase(t *testing.T) {
 	require.NotNil(t, conn2)
 	require.Equal(t, 5, len(conn2.Path.PathSegments))
 }
+
+func TestNSMGR_PassThrough(t *testing.T) {
```

Comment:
Can you also add a test that covers a case when all nses are registered in one nsmgr?

Reply:
Done